src/js/components/Home/SpeedyScreens/BookScreen.scss

```diff
@@ -0,0 +1,42 @@
+.character{
+  height: 100%;
+  width: 100%;
+  padding-top: 15rem;
+  padding-bottom: 16rem;
+}
+.char-window{
+  background-color: #ccc;
+  min-height: 100%;
+}
+.character-page{
+  display: flex;
+  align-items: center;
+  background-color: #ccc;
+}
+.item-info{
+  display: flex;
+  align-items: center;
+}
+.book{
+  height: 100%;
+  width: 100%;
+}
+.char-info{
+  display: flex;
+  align-items: center;
+  color: #FFF;
+  background-color: #4e555b;
+}
+.btn-col{
+  padding-right: 0px;
+  display: flex;
+  align-items: center;
+}
+.btnCollapse{
```

Comment:
white spaces between declarations 
GC 
Empty line at the end of the file

Reply:
fixed thanks